to_srt.py

```diff
@@ -52,7 +52,7 @@ def append_subs(start, end, prev_content):
         append_subs(prev_time["start"], prev_time["end"], prev_content)
         prev_time = {"start": start, "end": end}
         prev_content = [content]
-    append_subs(start, end, prev_content)
+        append_subs(start, end, prev_content)

```

Comment:
Also the sub_lines generator might be empty if the line start with spaces before the "<p begin=".
I've changed to use a regular expression: 
line_re = re.compile(u"\s*<p\s*begin=")
sub_lines =  filter(line_re.match,text.split("\n"))
I'm a little rusty on python and regexp but I think it works fie